engines/scumm/gfx_gui.cpp

```diff
@@ -2031,6 +2031,9 @@ void ScummEngine::setMusicVolume(int volume) {
 		_mixer->setVolumeForSoundType(Audio::Mixer::kMusicSoundType, volume * 2);
 	ConfMan.setInt("music_volume", volume * 2);
 	ConfMan.flushToDisk();
+
+	if (_game.version < 7)
```

Comment:
Why is this synchronization only done only for earlier games?

Reply:
I added this since it turns out that since I made the GUI, non-Digital-iMUSE games (so < 7) were not updating the sound settings because... I forgot :-) This fixes the GUI volume bars.

Reply:
OK :) Please add a comment for future reference

Reply:
Thanks, will do now.

Reply:
Done 👌